### DISCUSSION TITLE
Welcome to spring-framework Discussions!

### DISCUSSION THREAD

**Discussion by akash-coded** [akash-coded/spring-framework#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

  📢 **Announce to your community** that Discussions is available! Go ahead and send that tweet, post, or link it from the website to drive traffic here.

  🔗 If you use issue templates, **link any relevant issue templates** such as questions and community conversations to Discussions. Declutter your issues by driving community content to where they belong in Discussions. If you need help, here's a [link to the documentation](https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser).

  ➡️ You can **convert issues to discussions** either individually or bulk by labels. Looking at you, issues labeled “question” or “discussion”.

**Comment by akratimishra22** [comment]
Hi @akash-coded, I am almost done with the ecommerce assignments but I have a few question.
I'm not sure how to do these as Order has a single Product. Let me know if I should update the schema for an order to have multiple products?
- Finding all products in an Order
- Finding the Orders with more than 1000 Rs in value (I can still do this for a single product in an order)

Also creating a product creates one more entry in the Category as we discussed in the call, so can you suggest what should I do to cater below requirements?
- Finding all products for a Category
- Finding the Category with no Products
